app/src/main/java/org/sdc/ttu/sdc_test_game_app/Login/LoginInterface.java

```diff
@@ -17,6 +18,7 @@
     interface Presenter{
 
         void login(@NonNull String username, @NonNull String password);
+        void createActivity(Intent activity);
```

Comment:
This method's arguments should be `Context`, not `Intent`, this way we keep the View as dumb as possible (i.e, it won't have to create an intent to pass to the presenter).